.travis.yml

```diff
@@ -26,6 +30,11 @@ before_script:
 
 script:
     - export PORTAL_HOME=$(pwd)
+    - export PYTHONPATH=$PORTAL_HOME/core/src/main/scripts
+    - cd $PORTAL_HOME/core/src/test/scripts/
+    - $PORTAL_HOME/core/src/test/scripts/./unit_tests_validate_data.py
+    - $PORTAL_HOME/core/src/test/scripts/./system_tests_validate_data.py
+    - cd $PORTAL_HOME
```

Comment:
As mentioned, this is good for now, but if it's possible to use a test runner such as nosetests w/o too many changes to the code, I think that would be cleaner. Also, a comment here would be great.